google/resource_compute_region_instance_group_manager.go

```diff
@@ -145,6 +148,14 @@ func resourceComputeRegionInstanceGroupManager() *schema.Resource {
 					},
 				},
 			},
+
+			"distribution_policy": &schema.Schema{
```

Comment:
Is there any reason we don't deal with 'distribution_policy' changes in the update method? 
We should probably account for somebody adding/removing zones from the policy, and deal with changes in this argument

Reply:
The API doesn't allow you to change the policy after creating an rIGM, so there's no way to actually update it.

Reply:
Then you should add "ForceNew: true" , which would require the resource to be re-created if somebody wants to change that argument. Otherwise it will be constantly identifying the change without being able to do anything about it

Reply:
Done!